client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/RequestLineEditForm.tsx

```diff
@@ -220,8 +229,8 @@ export const RequestLineEditForm = ({
             InputProps={{
               sx: { backgroundColor: theme => theme.palette.background.menu },
             }}
-            minRows={6}
-            maxRows={6}
+            minRows={6.2}
+            maxRows={6.2}
```

Comment:
I also wanted to make this dynamic to fit the size of neighbouring grid components. 
I couldn't figure out an easy way to do this, however.

I've left it hard coded as it previously was, but adjusted it to align heights for the new layout.

Reply:
not sure if that's possible - couldn't see how to do that with the multiline prop set on a TextField without setting rows / minRows / maxRows.
I'd suggest `rows={6.2}` though it needs a change to TextArea to prevent a console warning:

```
export const TextArea: FC<StandardTextFieldProps> = ({
  value,
  onChange,
  maxRows = 4,
  minRows = 4,
  InputProps,
  rows,
  ...props
}) => (
  <BasicTextInput
    sx={{ width: '100%' }}
    InputProps={{
      ...InputProps,
      sx: {
        backgroundColor: 'white',
        ...InputProps?.sx,
      },
    }}
    multiline
    value={value}
    onChange={onChange}
    maxRows={rows ? undefined : maxRows}
    minRows={rows ? undefined : minRows}
    rows={rows}
    {...props}
  />
);
```

and I've suggested this before.. but perhaps in an as yet unmerged PR? I can't find it anyway..